File: round 1/pro/problem1.java
import java.util.Scanner;
class problem1
{
public void main()
{
Scanner sc= new Scanner(System.in);
System.out.println("enter number");
long long n= sc.nextLongLong();
long c=0;
for(long long i=1; i<=n; i++)
{
if(n%i==0)
c++;
}
if(c==2)
System.out.println("YES");
else 
System.out.println("NO");
}
}

File: q/c/submissions/goldensnitch/problem1.c
#include<stdio.h>
int main()
{
   int a,i,f=0;
   //printf("Enter the number \n");
   scanf("%d",&a);
   for(i=2;i<=a-1;i++)
    {  
     if(a%i==0)
       f++;
     }
     if(f==0)
    printf("Yes");
     else
    printf("No");
    return 0;
}